src/sysadmin/audit.md

```diff
@@ -85,6 +163,22 @@ The audit can be configured using the _audit matrix_. The audit matrix represent
 * `audit.tracker`
 * `audit.aggregate`
 
+### Artemis
+[Apache ActiveMQ Artemis](https://activemq.apache.org/components/artemis/documentation/) is an open source project to build a multi-protocol, embeddable, very high performance, clustered, asynchronous messaging system. It has been part of DHIS2 since version 2.31 and used as a system to consume audit logs.
+
+By default, DHIS2 will start an embedded Artemis server, which is used internally by the application to store and access audit events.
+
+However, if you have already an Artemis server, you can connect to it from DHIS2 to send audit events, as described in our [official documentation](https://docs.dhis2.org/en/manage/performing-system-administration/dhis-core-version-master/installation.html#webapi_amqp_configuration): in this setup, audit events will flow from DHIS2 to the external Artemis system.
```

Comment:
Link simply to an anchor in the docs, rather than an absolute URL and the docs generator will identify the relevant page/section:
```suggestion
However, if you have already an Artemis server, you can connect to it from DHIS2 to send audit events, as described in our [official documentation](#webapi_amqp_configuration): in this setup, audit events will flow from DHIS2 to the external Artemis system.
```